wagtail/contrib/modeladmin/views.py

```diff
@@ -577,15 +576,14 @@ def get_queryset(self, request=None):
         ordering = self.get_ordering(request, qs)
         qs = qs.order_by(*ordering)
```

Comment:
@seb-b Do we also need to think about preserving ordering here? `WagtailBackendSearchHandler` will just order results by best match by default, but I think there is a way to toggle that behaviour - which we might want to do if custom ordering has been specified?

Reply:
This is what I was thinking of:
http://docs.wagtail.io/en/v2.5/topics/search/searching.html#custom-ordering